src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -97,11 +97,17 @@ public UserBook findByName(@PathVariable String name) {
     }
 
     @GetMapping("/birthdayAndName")
```

Comment:
You can use the root path "/", try to use a less descriptive paths

Reply:
fixed at last commit